Sources/MapboxCoreNavigation/EHorizon/ElectronicHorizonEdge.swift

```diff
@@ -1,48 +1,47 @@
 import Foundation
 import MapboxNavigationNative
 
-extension ElectronicHorizon {
+/**
+ An electronic horizon is a probable path (or paths) of a vehicle. This structure contains metadata about an edge in a routing graph. For example, an edge may represent a road segment between two intersections or between the two ends of a bridge. An edge may traverse multiple road objects, and a road object may be associated with multiple edges. The road network ahead of the user is represented as a tree of edges. Each intersection has outlet edges. In turn, each edge has a probability of transition to another edge, as well as details about the road segment that the edge traverses. You can use these details to influence application behavior based on predicted upcoming conditions.
+
+ During active turn-by-turn navigation, the user-selected route and its metadata influence the path of the electronic horizon determined by `RouteController`. During passive navigation (free-driving), no route is actively selected, so `PassiveLocationDataSource` will determine the most probable path from the vehicle’s current location. You can receive notifications about changes in the current state of the electronic horizon by observing the `Notification.Name.electronicHorizonDidUpdatePosition`, `Notification.Name.electronicHorizonDidEnterRoadObject`, and `Notification.Name.electronicHorizonDidExitRoadObject` notifications.
+
+ Use a `RoadGraph` object to get an edge with a given identifier.
+ */
+public struct ElectronicHorizonEdge {
```

Comment:
Maybe we should call it `RoadGraphEdge`? We have other entities with such semantics and this one seems to fit the same idea.

Reply:
I think the less breaking our changes are for a user, the better it will be. Renaming from `ElectronicHorizon.Edge` to `ElectronicHorizonEdge ` feels simpler for the end user IMHO.
I also considered not removing `ElectronicHorizon` at all and instead convert it to enum without cases so that it will serve as a namespace, but then decided not to go with this soulution and just remove `ElectronicHorizon` at all.

Reply:
Booth names are "breaking". I don't consider it a huge issue while the SDK is in beta. My concern is about naming itself, but I don't insist on `RoadGraphEdge`.

Reply:
Nesting the type under `RoadGraph` as `RoadGraph.Edge` would be pretty elegant.

Reply:
Renamed `ElectronicHorizonEdge` to `RoadGraph.Edge`